src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> To be precise, this is only for Windows and OSX - Linux will fall to linux-*.

Correct.  That is my thought.  Since we actually have `linux` assets, falling to `linux` makes sense.

The good thing about Windows and OSX is that they are more controlled and usually have high compatibility requirements.